### DISCUSSION TITLE
Getting KeyError: 'HOSTINSTALLDIR' when using pyodide-build

### DISCUSSION THREAD

**Discussion by ouiliame** [pyodide/pyodide#2119] [Q&A]
Hi,

I was following the Pyodide documentation and trying to build a Pyodide package for a Python library I wrote. However, I keep running into:

```
➜  ~ pyodide-build
Traceback (most recent call last):
  File "/usr/local/bin/pyodide-build", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.9/site-packages/pyodide_build/__main__.py", line 44, in main
    HOSTINSTALLDIR = os.environ["HOSTINSTALLDIR"]
  File "/usr/local/Cellar/python@3.9/3.9.9/Frameworks/Python.framework/Versions/3.9/lib/python3.9/os.py", line 679, in __getitem__
    raise KeyError(key) from None
KeyError: 'HOSTINSTALLDIR'
```

This occurs on both macOS 12.1 Monterey for Intel and M1 computers. I've also tried this out on an x64 Ubuntu 20.04 with CPython 3.9, and get the same error. Is there something I'm missing? Please advise.

**Comment by ryanking13** [comment]
When building, try `make` instead of calling `pyodide-build buildpkg` directly.
We are planning to make `pyodide-build` run independently, but for now it has to be used inside the Makefile.

**Comment by fridgerator** [comment]
@ouiliame I am also getting this error, did you happen to find a fix?
